test/tests/ver-1fa/ver-1fa.i

```diff
@@ -13,43 +17,47 @@ volumetric_heat = '${units 1e4 W/m^3}'
 []
 
 [Variables]
-  [temp]
+  # temperature parameter in the slab in K
+  [temperature]
     initial_condition = '${initial_temperature}'
   []
 []
 
 [Kernels]
   [heat]
     type = HeatConduction
-    variable = temp
+    variable = temperature
   []
   [heatsource]
     type = HeatSource
     function = volumetric_heat
-    variable = temp
+    variable = temperature
   []
   [HeatTdot]
     type = HeatConductionTimeDerivative
-    variable = temp
+    variable = temperature
   []
 []
 
 [BCs]
-  [lefttemp]
+  # The temerature on the right boundary of the slib is kept at 300 K
```

Comment:
```suggestion
  # The temperature on the right boundary of the slab is kept constant
```
not specifying the temperature here makes it more general and easier to maintain.